### PR TITLE
api-minimum-version-number call documentation

### PR DESCRIPTION
**Pull request by v03adk**

### CODE REVIEW

File: docs/calls/api-minimum-version-number.rst [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+.. _calls-apiminimumversionnumber-docs:
+
+API minimum version number
+===========
+
+Request
+-------
+
+``"api-minimum-version-number"`` returns minimum required API version for work.
```
Comment (schemar) [inline docs/calls/api-minimum-version-number.rst]: Better: "``"api-minimum-version-number"`` returns minimum required API version that must be used by any client."

File: docs/calls/api-minimum-version-number.rst [hunk 2]
```diff
@@ -0,0 +1,44 @@
+.. highlight:: js
+
+.. _calls-apiminimumversionnumber-docs:
+
+API minimum version number
+===========
+
```
Comment (schemar) [inline docs/calls/api-minimum-version-number.rst]: Please add an "important::" that states that you have to update the client if you are not using at least the API version returned by this call and that you should not let the user interact with the API.

You can see an example in the station post call. You should test it locally.

**Comment by v03adk** [comment]
done

**Comment by schemar** [comment]
Nice :tada: thanks

**Pull request merged**